interfaces.go

```diff
@@ -101,15 +102,15 @@ func (r Request) FromHTTPRequest(request *http.Request) Request {
 	r.QueryString = request.URL.RawQuery
 
 	// Body
-	if request.GetBody != nil {
-		if bodyCopy, err := request.GetBody(); err == nil && bodyCopy != nil {
-			body, err := ioutil.ReadAll(bodyCopy)
-			if err == nil {
-				r.Data = string(body)
-			}
+	if request.Body != nil {
+		bodyBytes, err := ioutil.ReadAll(request.Body)
+		_ = request.Body.Close()
```

Comment:
Just my curiosity, why did you add the `_ =` part?

---

According to https://godoc.org/net/http#Request:

> ```go 
> Request {
>     //...
>     // Body is the request's body.
>     //
>     // For client requests, a nil body means the request has no
>     // body, such as a GET request. The HTTP Client's Transport
>     // is responsible for calling the Close method.
>     //
>     // For server requests, the Request Body is always non-nil
>     // but will return EOF immediately when no body is present.
>     // The Server will close the request body. The ServeHTTP
>     // Handler does not need to.
>     Body io.ReadCloser
> ```

I might be missing something, but I think we don't need to close the body here, the Server will do.

Reply:
> why did you add the _ = part?

Just to let linter know that i don't want to handle error here, if Body reader already closed
> I might be missing something, but I think we don't need to close the body here, the Server will do.

Hmm, i wasn't sure about it. But we're overwriting the original body reader and server will close the new one, it seems like resource leak

Reply:
> we're overwriting the original body reader and server will close the new one, it seems like resource leak

You're right! Sorry, my mistake.